### ISSUE TITLE
Position Point dont show up in new Grafana Version

### ISSUE THREAD

**Issue by mennchen22** [pR0Ps/grafana-trackmap-panel#53]
I use this plugin a lot for data analytics.
Normally the truck position based on the selected timestamp within another graph is shown.
This feature will not work with the current Grafana version, only a version reset re-enabled this feature.

Can you provide an update to the plugin, that will fix that?

Example of the use case:
![grafik](https://user-images.githubusercontent.com/32162894/108702594-c5e1d800-7509-11eb-9981-c9af72f149ec.png)

**Comment by pR0Ps** [comment]
Duplicate of #50

It's already been fixed and released, just waiting on Grafana to publish it (see #52)
